sources/platform/actors/development/actor_definition/actor_json.md

```diff
@@ -0,0 +1,51 @@
+---
+title: actor.json
+description: Learn how to write the main actor config in the `.actor/actor.json` file.
+slug: /actors/development/actor-config
+sidebar_position: 1
+---
+
+**Learn how to write the main actor config in the `.actor/actor.json` file.**
+
+---
+
+The main actor config should live in the `.actor/actor.json` file in the actor's root directory. This file associates your local development project with an actor on the Apify platform. It contains information such as actor name, version, build tag, and environment variables. Make sure you commit this file to the Git repository.
+
+For example, the `.actor/actor.json` file can look like this:
+
+```json
```

Comment:
I think it would be nice to use code tabs here and show a "minimal actor.json" and "full actor json". When you copy paste this, it's not clear what you can delete unless you read the docs below.

Reply:
I am moving this into the following round of changes, as this text was already there and was just move to a different location.